pkg/providers/aws/provider_postgres.go

```diff
@@ -552,35 +552,43 @@ func (p *PostgresProvider) deleteRDSInstance(ctx context.Context, pg *v1alpha1.P
 		return croType.StatusMessage(fmt.Sprintf("deletion protection detected, modifyDBInstance() in progress, current aws rds status is %s", *foundInstance.DBInstanceStatus)), nil
 	}
 
-	// standaloneNetworkExists if no bundled resources are found in the cluster vpc
-	if standaloneNetworkExists && isLastResource {
-		logger.Info("found the last instance of types postgres and redis so deleting the standalone network")
-		networkPeering, err := networkManager.GetClusterNetworkPeering(ctx)
+	// isEnabled is true if no bundled resources are found in the cluster vpc
+	if isEnabled && isLastResource {
+		saVPC, err := getStandaloneVpc(ctx, p.Client, ec2Svc, logger)
```

Comment:
do we need to handle not found error here?

Reply:
If you're meaning that the standaloneVPC isn't found then there won't be any error thrown and standaloneVPC will be nil. This is where we hit the nil pointer exception before. Instead of erroring here we assume that as we're in the delete portion of the code and the standalone isn't found (and we were supposed to deploy a standalone), either it was re-tagged so we couldn't find it, it's been deleted manually, or it was never created - in all three cases we skip cleaning up the associated networking resources and move onto removing the related CR.

After thinking a bit more about it I can't really imagine a situation where there is no VPC in the region for the describe VPC function to actually return an error - wouldn't that mean there's no standalone, no default, no cluster vpc, how would CRO even be running?